sentinel-demo/sentinel-demo-pigeon/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>sentinel-demo</artifactId>
+        <groupId>com.alibaba.csp</groupId>
+        <version>1.6.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>sentinel-demo-pigeon</artifactId>
+
+    <repositories>
+        <repository>
```

Comment:
Maybe it's not necessary to add the snapshot repository here?

Reply:
I have removed snapshot.

Reply:
Why is `github-dianping-maven-repo` is needed here? I've tried the URL and it's 400 bad request.

Reply:
Because pigeon dependency can be fetched only from `github-dianping-maven-repo`.

Reply:
> Why is `github-dianping-maven-repo` is needed here? I've tried the URL and it's 400 bad request.

You can compiled the root of sentinel, it's successful.